x/delegation/types/genesis.go

```
@@ -215,28 +218,38 @@
-		if undelegation.CompleteBlockNumber < undelegation.BlockNumber {
+		if undelegation.UndelegationId >= gs.LastUndelegationId {
 			return errorsmod.Wrapf(
-				ErrInvalidGenesisData, "the block number to complete shouldn't be less than the submitted , undelegation：%v",
-				undelegation,
+				ErrInvalidGenesisData, "the undelegationID should be less than the global undelegationID,undelegationID:%d,globalID:%d",
+				undelegation.UndelegationId, gs.LastUndelegationId,
 			)
 		}
-		if undelegation.ActualCompletedAmount.GT(undelegation.Amount) {
+		if undelegation.CompletedEpochNumber < 0 {
+			return errorsmod.Wrapf(
```

golangci-lint (1.62.2)

printf: cosmossdk.io/errors.Wrapf format %s has arg undelegation.CompletedEpochNumber of wrong type int64

(govet)